portfolio/.settings/org.eclipse.core.resources.prefs

```diff
@@ -0,0 +1,3 @@
+eclipse.preferences.version=1
```

Comment:
gitignore .settings folder

Reply:
fixed in 1c5f413fe6c41160d8b05b41b46102e2d9df161b